src/dymaptic.GeoBlazor.Core/Scripts/jsBuilder.ts

```diff
@@ -727,9 +731,9 @@ export function buildJsQuery(dotNetQuery: DotNetQuery): Query {
         where: dotNetQuery.where ?? "1=1",
         spatialRelationship: dotNetQuery.spatialRelationship as any ?? "intersects",
         distance: dotNetQuery.distance ?? undefined,
-        units: dotNetQuery.units as any ?? null,
```

Comment:
https://developers.arcgis.com/javascript/latest/api-reference/esri-rest-support-Query.html#units

The default value is null per the API?